tesseract_core/sdk/tesseract.py

```diff
@@ -313,10 +315,21 @@ def _request(
             encoded_payload = None
 
         response = requests.request(method=method, url=url, json=encoded_payload)
-        response.raise_for_status()
-
         data = response.json()
 
+        if response.status_code == 422 and "detail" in data:
```

Comment:
I always struggle to remember all 4xx status codes, what do you think about this instead:
```suggestion
        if response.status_code == requests.codes.unprocessable_entity and "detail" in data:
```
?